runtime/gc_base/GCExtensions.hpp

```diff
@@ -392,4 +392,4 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	}
 };
 
-#endif /* GCEXTENSIONS_HPP_ */
+#endif /* GCEXTENSIONS_HPP_ */
```

Comment:
Why this line has been changed? Last non-empty line in the file should have line termination.